requirements-freqai-rl.txt

```diff
@@ -0,0 +1,8 @@
+# Include all requirements to run the bot.
+-r requirements-freqai.txt
+
+# Required for freqai-rl
+torch==1.12.1
+stable-baselines3==1.6.1
+gym==0.21
+sb3-contrib==1.6.1
```

Comment:
as extras in setup.py? (so they are also installed / installable via `pip install freqtrade[freqai-rl]` (`pip install freqtrade[freqai]` will/would work ;)